src/FsCheck/Gen.fs

```diff
@@ -330,14 +343,38 @@ module Gen =
     ///Sequence the given list of generators into a generator of a list.
     //[category: Creating generators from generators]
     [<CompiledName("Sequence"); CompilerMessage("This method is not intended for use from F#.", 10001, IsHidden=true, IsError=false)>]
-    let sequenceToSeq generators = 
-        generators |> sequence |> map List.toSeq
+    let sequenceToSeq<'T, 'TSource when 'TSource :> IEnumerable<Gen<'T>>> (generators: 'TSource) =
```

Comment:
This function used to only take a single type argument. So this unfortunately breaks compatibility.